details/impersonation.md

```diff
@@ -75,7 +77,7 @@ In the impersonation case, it looks pretty much the same. The difference being t
 
 **⚠️ IMPORTANT:** If one or more of the external services are not mocked, ROR might inform Kibana that the impersonation is not supported. It's better to always define all mocks, to avoid the "Impersonation not supported" Elasticsearch response.
 
-How to define mocks in Kibana is described [here](../examples/impersonation/external-services-mocks-ui.md).
+ROR Kibana plugin obviously helps admins to define mocks in the easy way. The [detailed instruction](../examples/impersonation/external-services-mocks-ui.md) describes how to do that. 
```

Comment:
Another good idea in docs is to stay away from adverbs like obviously, simply, just, easily, used in absolute way:

❌ "to accomplish XYZ, <simply|just|obviously> do this" <-  (makes assumptions of what SHOULD be easy for the reader: it's judgemental and not informative).

✅ "An easier way to accomplish YQW would be using this tool..."   <-- (something is easier than something else, OK) 

Would rewrite this to:

"Follow the detailed instructions on how admins can [define  and configure services mocks]() using the dedicated graphical user interface".

Reply:
the same violation here (as I mentioned, for me it shows that [the guide](https://www.w3.org/QA/Tips/noClickHere) at least is not intuitive)

Reply:
Yep.

"ROR Kibana plugin helps administrators to visually create and edit service mocks with a dedicated graphical UI. Follow our [service mock configuration guide](.....) for more. "